src/components/cta/ExternalProvider.tsx

```diff
@@ -1,16 +1,26 @@
-import { FunctionComponent, PropsWithChildren, ReactNode, useEffect } from 'react'
+import { FunctionComponent, PropsWithChildren, Provider, ReactNode, useEffect } from 'react'
```

Comment:
Also lets fix this error that is causing the yarn lint fail, "'Provider' is defined but never used"